### PR TITLE
Ignore stale notification queues from notification.xml

### LINKED ISSUE #7650: notification: removing a notification target after enabling it makes the notification unusable
## Expected Behavior
When I remove a notification target from the config that was enabled earlier using `mc event add`, I should be able to work normally with Minio server, e.g: adding other notification targets and enabling them

## Current Behavior

Removing an enabled notification causes errors and make notification unusable by the user

```
API: GetBucketNotification(bucket=testbucket)
Time: 21:58:59 CET 05/15/2019
RequestID: 159EF6DA80245361
RemoteHost: 192.168.100.143
UserAgent: Minio (linux; amd64) minio-go/v6.0.21 mc/2019-05-14T15:31:34Z
Error: ARN 'arn:minio:sqs::1:webhook' not found
       3: cmd/notification.go:943:cmd.readNotificationConfig()
       2: cmd/bucket-notification-handlers.go:75:cmd.objectAPIHandlers.GetBucketNotificationHandler()
       1: net/http/server.go:1995:http.HandlerFunc.ServeHTTP()
```

## Possible Solution
Notifications that are enabled using mc event add and not found in config.json should be automatically removed

## Steps to Reproduce (for bugs)
1. `minio server /tmp/fs/`
2. `mc mb myminio/testbucket/`
3. `mc admin config get myminio/ >/tmp/config.json`
4. Add a webhook notification in /tmp/config.json
`"webhook":{"1":{"enable":true,"endpoint":"http://localhost/webhook/"}}`
5. `mc admin config set myminio </tmp/config.json`
6. `mc admin service restart myminio/`
7. `mc event add myminio/testbucket/ arn:minio:sqs::1:webhook`
8. Edit /tmp/config.json to rename "1" webhook notification to "2"
`"webhook":{"2":{"enable":true,"endpoint":"http://localhost/webhook/"}}`
9. `mc admin config set myminio </tmp/config.json`
10. `mc admin service restart myminio/`
11. `mc event add myminio/testbucket/ arn:minio:sqs::2:webhook`

## Context
One issue in UX when playing with notification targets in config

## Regression
No

## Your Environment
All

### PR DESCRIPTION
**Pull request by Praveenrajmani**
## Description
If the ARN does not exist in the targetlist, remove it from the notification.xml.

## Motivation and Context
Fixes #7650

## Regression
No

## How Has This Been Tested?
The steps are provided in #7650 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#e5d65ecc-ac8a-4ed7-a69a-24db2cb8648d) |
<a name="e5d65ecc-ac8a-4ed7-a69a-24db2cb8648d"></a>
### 7673-9710266/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:32669
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 06240a57eca8:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 10 minutes and 51 seconds
(4/14) Running aws-sdk-ruby tests ... FAILED in 4 minutes and 11 seconds
{
  "name": "aws-sdk-ruby",
  "function": "copyObject(source_bucket_name, target_bucket_name, source_file_name, target_file_name = )",
  "args": {
    "source_bucket_name": "aws-sdk-ruby-bucket-0fb8285c07a6",
    "target_bucket_name": "aws-sdk-ruby-bucket-20678fd5614a",
    "source_file_name": "datafile-1-kB",
    "target_file_name": "datafile-1-kB"
  },
  "duration": 242573.43,
  "error": "Net::ReadTimeout",
  "status": "FAIL"
}

Executed 3 out of 14 tests successfully.
```

**Review by sinhaashish — COMMENTED** [review]
Tested and its working . But IMO User should be prompted about the stale entries, before removing.

**Comment by Praveenrajmani** [comment]
I discussed with @balamurugana on this issue.. We should not try to delete/modify the notification.xml. Since there can be multiple users using those targets. And by removing as an admin during restart, We leave those users un-informed about this. There is a potential risk here that those users will still be believing that these targets work fine. The `ARN Not Found` is a valid error in case of any stale notification config setting. But we can still allow such setting by fixing the GetBucketNotification. ie, To make `mc config set / service restart` and `mc event list` to work in this case. By ignoring and continuing after printing the error response in the console.

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#11ebd38c-28e2-441a-9ab1-00f8ea3d526e) |
<a name="11ebd38c-28e2-441a-9ab1-00f8ea3d526e"></a>
### 7673-f061e2f/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:31139
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 22fa723d01e5:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-epmnawrhwuiiwy",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 360025,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15A0A3B72DA15DA8, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by Praveenrajmani** [comment]
Can i get some reviews @sinhaashish @balamurugana ?

File: cmd/bucket-notification-handlers.go [hunk 1]
```diff
@@ -71,30 +72,21 @@ func (api objectAPIHandlers) GetBucketNotificationHandler(w http.ResponseWriter,
...
+	configFile := path.Join(bucketConfigPrefix, bucketName, bucketNotificationConfig)
+	configData, err := readConfig(ctx, objAPI, configFile)
 	if err != nil {
-		// Ignore errNoSuchNotifications to comply with AWS S3.
-		if err != errNoSuchNotifications {
+		if err != errConfigNotFound {
+			writeErrorResponse(ctx, w, toAPIError(ctx, err), r.URL, guessIsBrowserReq(r))
+			return
+		}
+		configData, err = xml.Marshal(&event.Config{})
```
Comment (balamurugana) [inline cmd/bucket-notification-handlers.go]: you would need to set `XMLNS`

File: cmd/bucket-notification-handlers.go [hunk 2]
```diff
@@ -142,9 +134,10 @@ func (api objectAPIHandlers) PutBucketNotificationHandler(w http.ResponseWriter,
 		if event.IsEventError(err) {
 			apiErr = toAPIError(ctx, err)
 		}
-
-		writeErrorResponse(ctx, w, apiErr, r.URL, guessIsBrowserReq(r))
-		return
+		if _, ok := err.(*event.ErrARNNotFound); !ok {
```
Comment (balamurugana) [inline cmd/bucket-notification-handlers.go]: Why is unavailable ARN supported for `Put Bucket Notification`?

Comment (balamurugana) [inline cmd/bucket-notification-handlers.go:156]: Why is unavailable ARN supported for `Put Bucket Notification`?

File: pkg/event/config.go [hunk 3]
```diff
@@ -272,15 +272,14 @@ func (conf *Config) ToRulesMap() RulesMap {
...
+	if err = xml.NewDecoder(reader).Decode(&config); err != nil {
 		return nil, err
 	}
 
-	config.SetRegion(region)
+	err = config.Validate(region, targetList)
 
-	return &config, nil
+	config.SetRegion(region)
+	return &config, err
```
Comment (balamurugana) [inline pkg/event/config.go:294]: What does it mean to return non-nil config and error?

File: cmd/bucket-notification-handlers.go (hunk 2 above)
Comment (Praveenrajmani) [reply cmd/bucket-notification-handlers.go:156]: Previously, we would not save the notification config even if one of the target ARNs is not found. By ignoring this ARN not found error we would save the remaining ARNs atleast in the config

File: pkg/event/config.go (hunk 3 above)
Comment (Praveenrajmani) [reply pkg/event/config.go:294]: Now that we should not send an nil config for an `ARN Not Found` error. the config will not be empty/nil any more. By this way we would always save the config even if one of the ARN is not found

**Comment by Praveenrajmani** [comment]
Addressed the review comments. The PR is ready for review @balamurugana  @sinhaashish

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#123c9ce5-b9ce-42f1-b749-d3a972e5dd52) |
<a name="123c9ce5-b9ce-42f1-b749-d3a972e5dd52"></a>
### 7673-8257106/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:31343
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 3a77afe7abb2:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-12gcc4c1vfladf",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 3061,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:31343/aws-sdk-go-test-12gcc4c1vfladf: dial tcp 72.28.97.57:31343: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by Praveenrajmani** [comment]
PTAL @balamurugana @sinhaashish

**Review by sinhaashish — APPROVED** [review]
LGTM

**Comment by kannappanr** [comment]
Can you please review @balamurugana

**Comment by Praveenrajmani** [comment]
~I need to test this once before merging~. Tested. PTAL @balamurugana

File: cmd/bucket-notification-handlers.go (hunk 1 above)
Comment (Praveenrajmani) [reply cmd/bucket-notification-handlers.go]: done :+1:

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#d58d4c81-4f24-4455-bf43-630a9a75b611) |
| mint-large-bucket.sh | :x: [more...](#94179d44-564a-4b84-b5df-ea675946838f) |
| mint-compression-fs.sh | :x: [more...](#e6ed99b0-4fe7-42ce-978a-94f9656166ab) |
| mint-gateway-nas.sh | :x: [more...](#cb24c736-aa70-4efe-829b-c62d4de19f06) |
<a name="cb24c736-aa70-4efe-829b-c62d4de19f06"></a>
### 7673-dbc2ff8/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:30987
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp de4a20dadc90:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 18 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-5h2llnd244gopi",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 17982,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:30987/aws-sdk-go-test-5h2llnd244gopi: dial tcp 72.28.97.57:30987: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="d58d4c81-4f24-4455-bf43-630a9a75b611"></a>
### 7673-dbc2ff8/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:32463
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e5c6d63b1c65:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-k0m01wz3ru1fsr",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 360012,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15B4E28E1439C9FE, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="94179d44-564a-4b84-b5df-ea675946838f"></a>
### 7673-dbc2ff8/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:30944
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 9a1fdb8c5a55:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 47 seconds
(4/14) Running aws-sdk-ruby tests ... done in 14 seconds
(5/14) Running awscli tests ... done in 2 minutes and 23 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 32 seconds
{
  "name": "mc",
  "duration": "126134",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-26040 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-18918  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-18918/datafile-1-MB`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-18918/datafile-1-b`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-18918/datafile-0-b`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-18918/datafile-1-kB`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-18918/datafile-1.03-MB`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-18918/datafile-10-kB`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-18918/datafile-10-MB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-18918/datafile-11-MB`\n`/mint/data/datafile-129-MB` -> `myminio/mc-test-bucket-18918/datafile-129-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-18918/datafile-100-kB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-18918/datafile-5-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-18918/datafile-33-kB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-18918/datafile-5243880-b`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-18918/datafile-65-MB`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-18918/datafile-6-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-129-MB`. Please reduce your request\nTotal: 233.17 MiB, Transferred: 382.49 MiB, Speed: 3.03 MiB/s"
}

Executed 6 out of 14 tests successfully.
```
<a name="e6ed99b0-4fe7-42ce-978a-94f9656166ab"></a>
### 7673-dbc2ff8/mint-compression-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:32746
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d9bea2c4adbc:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 16 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-iv36bdhv9y542e",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 15365,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:32746/aws-sdk-go-test-iv36bdhv9y542e: dial tcp 72.28.97.57:32746: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#f1d3a886-06e8-425b-95d0-b599fb6fd09d) |
| mint-worm.sh | :x: [more...](#41092289-4261-4ca0-87fb-22961d67915e) |
| mint-dist-xl.sh | :x: [more...](#aaf48829-321e-48e1-9347-32d1cb911d7e) |
<a name="f1d3a886-06e8-425b-95d0-b599fb6fd09d"></a>
### 7673-e932bd1/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:30576
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 34cba4ddacec:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 2 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 52 seconds
(4/14) Running aws-sdk-ruby tests ... done in 18 seconds
(5/14) Running awscli tests ... done in 1 minutes and 29 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 18 seconds
{
  "name": "mc",
  "duration": "100260",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-21399 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-24331  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-24331/datafile-1-MB`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-24331/datafile-0-b`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-24331/datafile-1-kB`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-24331/datafile-10-MB`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-24331/datafile-1.03-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-24331/datafile-100-kB`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-24331/datafile-1-b`\n`/mint/data/datafile-129-MB` -> `myminio/mc-test-bucket-24331/datafile-129-MB`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-24331/datafile-10-kB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-24331/datafile-11-MB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-24331/datafile-5-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-24331/datafile-33-kB`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-24331/datafile-6-MB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-24331/datafile-5243880-b`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-24331/datafile-65-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-11-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-129-MB`. Please reduce your request\nTotal: 233.17 MiB, Transferred: 434.61 MiB, Speed: 4.34 MiB/s"
}

Executed 6 out of 14 tests successfully.
```
<a name="41092289-4261-4ca0-87fb-22961d67915e"></a>
### 7673-e932bd1/mint-worm.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:31373
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            worm
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp da401971edd0:/mint/log /tmp/mint-logs'

(1/1) Running worm tests ... FAILED in 12 seconds
{
  "alert": "",
  "args": {
    "bucketName": "bucket-seiwn7leqfz6jxdwvbhz66gthv9n8jo9j9hx6lopiobsxqlws341bra",
    "expiry": 60000000000,
    "objectName": "testObject"
  },
  "duration": 12513,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:31373/bucket-seiwn7leqfz6jxdwvbhz66gthv9n8jo9j9hx6lopiobsxqlws341bra: dial tcp 72.28.97.57:31373: connect: no route to host",
  "function": "PutAndDelete",
  "message": "WORM_MODE ON  - CreateBucket Failed",
  "name": "test worm mode",
  "status": "FAIL"
}

Executed 0 out of 1 tests successfully.
```
<a name="aaf48829-321e-48e1-9347-32d1cb911d7e"></a>
### 7673-e932bd1/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:32209
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 0cd2baaec16c:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 2 minutes and 4 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 1 minutes and 15 seconds
(8/14) Running minio-dotnet tests ... done in 34 seconds
(9/14) Running minio-go tests ... done in 50 seconds
(10/14) Running minio-java tests ... FAILED in 47 seconds
{
  "name": "minio-java",
  "function": "listIncompleteUploads(String bucketName, String prefix)",
  "duration": 1202,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-2p8v4g6, objectName=minio-java-test-2oi0bq2, resource=/minio-java-test-2p8v4g6/minio-java-test-2oi0bq2, requestId=15B4E41A1D381F9A, hostId=d56e6d9b-5a52-4239-be3b-2d48add7092f)\nrequest={method=DELETE, url=http://72.28.97.55:32209/minio-java-test-2p8v4g6/minio-java-test-2oi0bq2?uploadId=129f2e42-f02d-4e41-a2e5-5a24a896fb02, headers=Host: 72.28.97.55:32209\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20190726T072251Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20190726/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 363\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-07-26T07-03-44Z\nVary: Origin\nX-Amz-Request-Id: 15B4E41A1D381F9A\nX-Xss-Protection: 1; mode=block\nDate: Fri, 26 Jul 2019 07:22:52 GMT\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1070), io.minio.MinioClient.executeDelete(MinioClient.java:1347), io.minio.MinioClient.abortMultipartUpload(MinioClient.java:5149), io.minio.MinioClient.putObject(MinioClient.java:4337), io.minio.MinioClient.putObject(MinioClient.java:4183), FunctionalTest.listIncompleteUploads_test2(FunctionalTest.java:1639), FunctionalTest.runTests(FunctionalTest.java:2989), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```

**Review by balamurugana — APPROVED** [review]

**Pull request merged**